modules/distribution/src/assembly/bin.xml

```diff
@@ -319,200 +300,100 @@
+        </fileSet>
 
         <!-- Copying Device Management related dbscripts -->
         <fileSet>
-            <directory> ../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/dbscripts/cdm</directory>
+            <directory>
+                ../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/dbscripts/cdm
+            </directory>
             <outputDirectory>${pom.artifactId}-${pom.version}/dbscripts/cdm</outputDirectory>
             <includes>
                 <include>*/**</include>
             </includes>
         </fileSet>
         <fileSet>
-            <directory>src/statistics/carbonapps/target</directory>
-            <outputDirectory>${pom.artifactId}-${pom.version}/repository/deployment/server/carbonapps
+            <directory>
+                ../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/resources
+            </directory>
+            <outputDirectory>${pom.artifactId}-${pom.version}/repository/resources
             </outputDirectory>
-            <fileMode>755</fileMode>
         </fileSet>
 
+
+        <!-- Copying Mobile plugin management related dbscripts -->
         <fileSet>
             <directory>
-                ../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/deployment/server/jaggeryapps
+                ../p2-profile-gen/target/wso2carbon-core-${carbon.mobile.device.mgt.version}/dbscripts/cdm/plugins
             </directory>
-            <outputDirectory>${pom.artifactId}-${pom.version}/repository/deployment/server/jaggeryapps/
+            <outputDirectory>${pom.artifactId}-${pom.version}/dbscripts/cdm/plugins
             </outputDirectory>
-            <excludes>
-                <exclude>**/publisher/config/publisher-tenant.json</exclude>
-                <exclude>**/sso/jaggery.conf</exclude>
-                <exclude>**/sso/views/register_new_user.jag</exclude>
-                <exclude>**/devicemgt/*</exclude>
-            </excludes>
+            <includes>
+                <include>*/**</include>
+            </includes>
         </fileSet>
 
-
-
-        <!-- Copying mdm app from devicemgt app-->
+        <!-- Copying Authentication Endpoint webapp -->
         <fileSet>
             <directory>
-                ../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/deployment/server/jaggeryapps/devicemgt/
+                ../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/deployment/server/webapps
             </directory>
+            <outputDirectory>${pom.artifactId}-${pom.version}/repository/deployment/server/webapps
+            </outputDirectory>
+            <includes>
+                <include>authenticationendpoint.war</include>
+                <include>oauth2.war</include>
+                <include>dynamic-client-web.war</include>
+            </includes>
+        </fileSet>
+        <!--copy jaxrs web app for OAuth feature-->
+
+
```

Comment:
Already fixed in commit https://github.com/wso2-incubator/product-iot-server/commit/febdc36cba8b94c3ebcaba27947893ebb2865513